src/tests/test_sexual_behaviour.py

```diff
@@ -104,3 +106,225 @@ def test_initial_sex_behaviour_groups():
             Expectation = float(n_sex)*p
             N_g = sum(index_sex & index_group)
             assert Expectation - sigma*3 <= N_g <= Expectation + sigma*3
+
+
+def test_rred_long_term_partner():
+    N = 1000
+    pop = Population(size=N, start_date=date(1989, 1, 1))
+    # pick some indices and give those people LTPs
+    indices = [randint(0, N-1) for i in range(15)]
+    pop.data["partnered"] = False
+    pop.data.loc[indices, "partnered"] = True
+    SBM = SexualBehaviourModule()
+    SBM.update_rred_long_term_partnered(pop.data)
+    for i in indices:
+        assert pop.data.loc[i, "rred_long_term_partnered"] == SBM.ltp_risk_factor
```

Comment:
Could also compare in one go:
```suggestion
    assert all(pop.data.loc[i, "rred_long_term_partnered"] == SBM.ltp_risk_factor)
```

Reply:
I assume here `i` --> `indices` ?